lib/tasks/sul.rake

```diff
@@ -59,27 +59,39 @@ namespace :sul do
   end
 
   desc 'Export publications as csv'
-  # bundle exec rake sul:export_pubs_csv['/tmp/output_file.csv','01/01/2013'] # parameters are output csv file, and date to go back to in format of mm/dd/yyyy
-  task :export_pubs_csv, %i[output_file date_since] => :environment do |_t, args|
+  # bundle exec rake sul:export_pubs_csv['/tmp/output_file.csv','01/01/2013','01/01/2014']
+  # parameters are output csv file, start date and end date in format of mm/dd/yyyy
+  task :export_pubs_csv, %i[output_file date1 date2] => :environment do |_t, args|
```

Comment:
I wonder if you think it's worth considering changing the `output_file`, `date1`, and `date2` argument names so that their use is clearer throughout the task definition, which may also obviate the need for the code comments?

Reply:
Sure, I'm changing the `date1` and `date2` but not sure about `output_file` ...